php-nginx/whitelist_functions.php

```diff
@@ -14,22 +14,33 @@
     $whitelist = array_filter(array_map('trim', explode(',', $whitelist)));
 }
 
-$suhosinBlacklist = ini_get('suhosin.executor.func.blacklist');
 $disable_functions = ini_get('disable_functions');
 
+// It's tricky, but we still don't have suhosin enabled and we can not fetch
+// the list from our suhosin config. Now it defines the list as a literal.
+// Note: We need to sync this list with our default suhosin config
+// (deb-package-builder/extensions/suhosin/debian/ext-suhosin.ini).
+// TODO: Find a better way
```

Comment:
It's not easily possible because composer fails when suhosin is enabled. We can add some flags to workaround that (like we did long tim ago)though. Also I think it's safer to run the script without enabling suhosin.